sshuttle/client.py

```diff
@@ -572,9 +572,9 @@ def main(listenip_v6, listenip_v4,
          user, sudo_pythonpath):
 
     if not remotename:
-        # XXX: We can't make it required at the argparse level,
-        #      because sshuttle calls out to itself in FirewallClient.
-        raise Fatal("You must specify -r/--remote.")
+        print("WARNING: You must specify -r/--remote to securely route "
```

Comment:
Won't this print a confusing message when sshuttle starts itself for the firewall?

Reply:
This warning is printed in the client program's main() function which does not get run when the --firewall options is supplied. If --firewall is given, the firewall's main() function is executed instead.

https://github.com/sshuttle/sshuttle/blob/a266e7a8bdb492a3a0d4af26944c25da9dfac43f/sshuttle/cmdline.py#L42-L46